internal/cmd/commands/connect/funcs.go

```diff
@@ -114,7 +114,8 @@ func generateConnectionInfoTableOutput(in ConnectionInfo) string {
 	var ret []string
 
 	nonAttributeMap := map[string]any{
-		"Connections Left": in.ConnectionsLeft,
+		"Connections Left":  in.ConnectionsLeft,
+		"Connections Count": in.ConnectionsCount,
```

Comment:
This is completely optional. It make it easy for me to debug stuff so I left it in. Happy to take it out